static/css/impala/devhub-api.less

```diff
@@ -12,8 +12,6 @@
     label {
       font-weight: bold;
       display: block;
-      width: 10%;
```

Comment:
You can remove `width: 78%` from `input[type=text]` above as well, so it goes full-width and the user can see their API key in full 😄